charts/grafana/values.yaml

```diff
@@ -195,9 +196,10 @@ gossipPortName: gossip
 ## or access it from within the cluster (ClusterIP service). Set the service type and the port to serve it.
 ## ref: http://kubernetes.io/docs/user-guide/services/
 ##
+
 service:
   enabled: true
-  type: ClusterIP
+  type: LoadBalancer
```

Comment:
I don't think it's a good idea changing default from ClusterIP to LoadBalancer, because:
- The user can change their values.yaml during the installation. For example,
  - `--set service.type=LoadBalancer`
  - `helm template grafana . -f /path/to/values.yaml -f /path/to/values_override.yaml`
  - `helm install grafana . -f /path/to/values.yaml -f /path/to/values_override.yaml`
- `LoadBalancer` option doesn't represent most majority, out-of-the-box use case. Also hard to test the load balancer and hard to separate the problem once malfunction.

Reply:
I see seems like that's the reason why lint test kept failing because most of the use cases did not supported ```LoadBalancer``` as default.

I have defined options for ```LoadBalancer``` service as it was stated in the ```service.yaml``` if [check](https://github.com/grafana/helm-charts/blob/acd598f80f30ce04302ad2487e572cec09e8fd67/charts/grafana/templates/service.yaml#L23C1-L32C14), please have a look if I got it right this time.